src/app/manufacturing/substrateFeatureMap.ts

```diff
@@ -0,0 +1,57 @@
+import Feature from "../core/feature";
+
+/**
+ * Substrate feature map class
+ */
+export default class SubstrateFeatureMap {
+    /**
+     * Default Constructor for DepthFeatureMap object.
+     * @param {string} name
+     */
```

Comment:
@MurrayLF  Would it make sense to merge depthfeaturemap and substratefeaturemap ?

Reply:
I've removed substrate map as it is no longer being used. If we change laser cutting to be more like the new format in CNC, then depth map will likely no longer be needed as well.